.changes/nextrelease/fix-closes-internal-stream.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "type": "feature",
+        "category": "Api",
```

Comment:
I would leave this field blank since the `Middleware` class doesn't belong to that namespace